Trimble.Modus.Components/Enums/ButtonColor.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Trimble.Modus.Components.Enums
+{
+    public enum ButtonColor
+    {   
+        None ,
```

Comment:
What is the use of `None` here? `Primary` should be the first and the default one